spec.bs

```diff
@@ -175,25 +175,25 @@ For any [=shape tree=] <code>S</code> linked in <code>ST</code>:
     indicates that there is exactly one corresponding resource <code>R</code>
     and it has the name <code>L</code>.
 * An { <code>S</code> <code class="vocab">st:matchesUriTemplate</code>
-    <code>L</code> } arc indicates that there <em class="rfc2119">MAY</em>
+    <code>L</code> } [=arc=] indicates that there <em class="rfc2119">MAY</em>
     be an arbitrary number of corresponding resources <code>R</code>
     and each will have a name that matches the URI template
     [[RFC6570]] <code>L</code>.
 * An { <code>S</code> <code class="vocab">st:validatedBy</code> <code>Sh</code> }
-    arc indicates that <code>R</code> <em class="rfc2119">MUST</em>
-    have exactly one node which conforms to shape <code>Sh</code>.
+    [=arc=] indicates that <code>R</code> <em class="rfc2119">MUST</em>
+    have exactly one node which conforms to [=shape=] <code>Sh</code>.
 * Any { <code>S</code> <code class="vocab">st:contains</code> <code>S2</code> }
-    arcs indicate there is a nested [=shape tree=] and corresponding nested
+    [=arcs=] indicate there is a nested [=shape tree=] and corresponding nested
     resource <code>R2</code>.
 * An { <code>S</code> <code class="vocab">st:references</code> <code>S3</code> }
-    arc indicates that [=shape tree=] <code>S3</code> is referenced through the
+    [=arc=] indicates that [=shape tree=] <code>S3</code> is referenced through the
     instance data for <code>S</code>.
 * An { <code>S</code> <code class="vocab">st:supports</code> <code>S4</code> }
-    arc indicates that <code>S</code> provides a different representation of
+    [=arc=] indicates that <code>S</code> provides a different representation of
     <code>S4</code> for aggregation purposes.
 
 A { <code class="vocab">st:hasShapeTreeDecoratorIndex</code> <code>Si</code> }
-    arc indicates the location of an index of SKOS hierarchies that describe
+    [=arc=] indicates the location of an index of [SKOS hierarchies](#skos-graph) that describe
```

Comment:
Doesn't seem to link to anything else in the document?

Reply:
Arc links to the arc definition.  SKOS hierarchies links to skos graph definition